core/src/main/java/bio/terra/pearl/core/model/fileupload/ParticipantFileUpload.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.pearl.core.model.fileupload;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class ParticipantFileUpload extends BaseEntity {
+    private String fileName;
+    private String fileType; // e.g. pdf, docx, etc.
+
+    private UUID uploadedFileId; // files are uploaded as UUIDs
+
+    private UUID creatingPortalParticipantUserId; // the user who uploaded the file; could be proxy
```

Comment:
I think these should be attached to enrollee and studyEnvironment (perhaps instead of portalParticipantUserId, but at least in addition to).   Participants upload files after consenting to a particular study, so we have to track which consent the files were uploaded in accordance with.  We'll eventually want to give enrollees the ability to easily share/propagate their files across multiple studies, but IIRC the default should always be study-scoped, and in the "file hub" we'll need to clearly indicate which study(ies) the files belong to